cmd/all-in-one/main.go

```diff
@@ -287,6 +288,10 @@ func startQuery(
 	if err := server.Start(); err != nil {
 		svc.Logger.Fatal("Could not start jaeger-query service", zap.Error(err))
 	}
+	if err = tracer.Close(context.Background()); err != nil {
```

Comment:
so when do you think this will be executed?

Reply:
Sorry! I think it was supposed to be here `svc.RunAndThen(func() {`